BookingsApi/BookingsApi.IntegrationTests/Database/Commands/AddJudiciaryPersonCommandTests.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Threading.Tasks;
+using BookingsApi.DAL;
+using BookingsApi.DAL.Commands;
+using BookingsApi.DAL.Queries;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace BookingsApi.IntegrationTests.Database.Commands
+{
+    public class AddJudiciaryPersonCommandTests : DatabaseTestsBase
+    {
+        private AddJudiciaryPersonCommandHandler _commandHandler;
+        private GetJudiciaryPersonByExternalRefIdQueryHandler _getJudiciaryPersonByExternalRefIdQueryHandler;
+
+        [SetUp]
+        public void Setup()
+        {
+            var context = new BookingsDbContext(BookingsDbContextOptions);
+            _commandHandler = new AddJudiciaryPersonCommandHandler(context);
+            _getJudiciaryPersonByExternalRefIdQueryHandler = new GetJudiciaryPersonByExternalRefIdQueryHandler(context);
+        }
+        
+        [Test]
+        public async Task should_add_person()
+        {
+            var externalRefId = Guid.NewGuid();
+            const string addText = "add";
+            var addCommand = new AddJudiciaryPersonCommand(externalRefId, addText, addText, addText, addText, addText, addText, addText);
+            await _commandHandler.Handle(addCommand);
+
+            var addedPerson = await _getJudiciaryPersonByExternalRefIdQueryHandler.Handle(new GetJudiciaryPersonByExternalRefIdQuery(externalRefId));
+
+            addedPerson.ExternalRefId.Should().Be(addCommand.ExternalRefId);
+            addedPerson.Email.Should().Be(addText);
```

Comment:
maybe for the test we could use values of "propertyName" so test all the params are in the correct order. I have checked manually in PR - they are but it would be a nice improvement to the test

Reply:
Done